modules/apps/static/portal-configuration/portal-configuration-persistence/build.gradle

```diff
@@ -1,8 +1,10 @@
 dependencies {
 	provided group: "com.liferay", name: "com.liferay.osgi.util", version: "3.0.0"
+	provided group: "com.liferay", name: "com.liferay.osgi.service.tracker.collections", version: "2.0.0"
 	provided group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "2.0.0"
 	provided group: "org.apache.felix", name: "org.apache.felix.configadmin", version: "1.8.8"
 	provided group: "org.osgi", name: "org.osgi.core", version: "5.0.0"
 	provided group: "org.osgi", name: "org.osgi.service.cm", version: "1.5.0"
 	provided group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
+	provided project(":apps:foundation:configuration-admin:configuration-admin-action")
```

Comment:
Just move them into that module. I'm not sure it's worth it creating a module for just those two classes.